src/azure-cli-tests/azure/cli/tests/checkers.py

```diff
@@ -0,0 +1,52 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import collections
+import jmespath
+from azure.cli.tests.exceptions import JMESPathCheckAssertionError
+
+
+class JMESPathCheck(object):
```

Comment:
The form changed. Now the comparing happens in a '__call__' so that a class can behave like a function therefor support checkers in both class and function form.